# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/bluez/hotfix-basename.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- bluez-5.72/tools/hex2hcd.c.vanilla	2024-07-25 12:51:33.372790445 +0200
+++ bluez-5.72/tools/hex2hcd.c	2024-07-25 12:51:41.261790350 +0200
@@ -24,6 +24,7 @@
 #include <stdlib.h>
 #include <stdbool.h>
 #include <sys/stat.h>
+#include <libgen.h>
 
 static ssize_t process_record(int fd, const char *line, uint16_t *upper_addr)
 {
